### PR TITLE
Upgrade MySQL.Data  v8.0.12

### PR DESCRIPTION
**Pull request by robvanpamel**
---
Rebus is [MIT-licensed](https://opensource.org/licenses/MIT). The code submitted in this pull request needs to carry the MIT license too. By leaving this text in, __I hereby acknowledge that the code submitted in the pull request has the MIT license and can be merged with the Rebus codebase__.

### CODE REVIEW

**Comment by robvanpamel** [comment]
Since .NET SDK 2.1.401 was released, Mysql.Data v7.0.7-m61 could not be referenced anymore.
"the package contains an entry which is unsafe for extraction." Updating to v8.0.12 resolves this issue.

File: Rebus.MySql.Tests/Rebus.MySql.Tests.csproj [hunk 1]
```diff
@@ -26,8 +26,8 @@
     <WarningLevel>4</WarningLevel>
     <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
   </PropertyGroup>
-  <PropertyGroup Condition=" '$(TargetFramework)' == 'net451' ">
-    <DefineConstants>NET451</DefineConstants>
+  <PropertyGroup Condition=" '$(TargetFramework)' == 'net452' ">
+    <DefineConstants>NET452</DefineConstants>
```
Comment (mookid8000) [inline Rebus.MySql.Tests/Rebus.MySql.Tests.csproj:30]: From this it looks like you intended to target .NET 4.5.2, but in the `<TargetFrameworks>` element you only added .NET Standard 1.6..... 🤔

**Comment by robvanpamel** [comment]
Ah I removed it to run it locally. ( Linux machine without .net4.5.2). You
can add it again. It wasn't supposed to be checked in.

Op vr 24 aug. 2018 22:22 schreef Mogens Heller Grabe <
notifications@github.com>:

**Comment by mookid8000** [comment]
Perfect 👍  it'll be out in Rebus.MySql 1.1.0 in a few minutes 😄

**Comment by robvanpamel** [comment]
Thanks!

Op zo 26 aug. 2018 om 09:46 schreef Mogens Heller Grabe <
notifications@github.com>

**Pull request merged**